regsync.yaml

```diff
@@ -611,6 +611,12 @@ sync:
 - source: dp.apps.rancher.io/containers/redis:8.0.3-2.1
   target: stgregistry.suse.com/rancher/appco-redis:8.0.3-2.1
   type: image
+- source: dp.apps.rancher.io/containers/suse-virtual-cluster-engine:1.0.0
```

Comment:
This is not correct.

I encourage to use the [image-lock file](https://apps.rancher.io/artifacts/2a83ac621b987549ab6f40007bb6a1d355fe287137c11afb12e8b406e2fdee15) associated to the Helm chart.

For SUSE Virtual Cluster Engine `1.0.0`:

- `dp.apps.rancher.io/containers/k3k:1.0.0-1.1`
- `dp.apps.rancher.io/containers/k3k-kubelet:1.0.0-1.1`